modules/oe_content_call_proposals/config/install/field.field.node.oe_call_proposals.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_call_proposals
+  module:
+    - text
+id: node.oe_call_proposals.body
+field_name: body
+entity_type: node
+bundle: oe_call_proposals
+label: 'Body text'
+description: ''
+required: false
+translatable: false
```

Comment:
This field is translatable according to requirements.

Reply:
This is fixed.